tasks/task876_emotion_esd.json

```diff
@@ -0,0 +1,644 @@
+{
+    "Contributors": [
+        "Archan Lagoo"
+    ],
+    "Source": [
+        "emotion (\"https://huggingface.co/datasets/emotion\")"
+    ],
+    "Definition": "This task involves detection of phrases in sentences that evoke a specific emotion when together. This is meant to imitate human attention when reading sentences. The input consists of a sentence (string) and an emotion label (string) as \"SENTENCE\nLABEL\", while the ouptut must contain a series of phrases (string) that evoke the emotion formatted as \"phrase\nphrase\n...\".The list of emotions includes sadness, joy, love, anger, fear, and surprise.",

```

Comment:
Does not add anything to the task definition: "This is meant to imitate human attention when reading sentences. "  drop it.